bandpass/README.md

```diff
@@ -194,6 +220,22 @@ Similarly, **f** can ge defined in terms of **alpha** and **delta_T**:
 
 f = alpha/((1-alpha)\*2\*pi\*delta_T)
 
+The script `lowpass_run.sql` runs the function on the
```

Comment:
Much better, thanks!

Question: do subsequent runs add to the _out tables or replace them?

Reply:
(Now answered in the text: replace tables)